src/containers/linking/EntityLinksContainer.js

```diff
@@ -0,0 +1,119 @@
+/*
+ * @flow
+ */
+
+import React, { Component } from 'react';
+
+import styled from 'styled-components';
+import { List, Map } from 'immutable';
+import { DataApiActions } from 'lattice-sagas';
+import { Card, Spinner } from 'lattice-ui-kit';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { RequestStates } from 'redux-reqseq';
+import type { RequestSequence, RequestState } from 'redux-reqseq';
+
+import { EntityLinkingCard } from './components';
+
+const { GET_LINKED_ENTITY_SET_BREAKDOWN } = DataApiActions;
+
+const CardGrid = styled.div`
+  display: grid;
+  grid-gap: 30px;
+  grid-template-columns: repeat(auto-fill, minmax(450px, 1fr));
+
+  ${Card} {
+    min-width: 0; /* setting min-width ensures cards do not overflow the grid column */
+  }
+`;
+
+type OwnProps = {|
+  entityKeyId :UUID;
+  entitySetId :UUID;
+|};
+
+type Props = {
+  ...OwnProps;
+  actions :{
+    getLinkedEntitySetBreakdown :RequestSequence;
+  };
+  entitySets :List;
+  entitySetsIndexMap :Map;
+  links :Map;
+  requestStates :{
+    GET_LINKED_ENTITY_SET_BREAKDOWN :RequestState;
+  };
+};
+
+class EntityLinksContainer extends Component<Props> {
+
+  componentDidMount() {
+
+    const { actions, entityKeyId, entitySetId } = this.props;
+    actions.getLinkedEntitySetBreakdown({ entitySetId, entityKeyIds: [entityKeyId] });
+  }
+
+  componentDidUpdate(props :Props) {
+
+    const { actions, entityKeyId, entitySetId } = this.props;
+
+    if (props.entityKeyId !== entityKeyId || props.entitySetId !== entitySetId) {
+      actions.getLinkedEntitySetBreakdown({ entitySetId, entityKeyIds: [entityKeyId] });
+    }
+  }
```

Comment:
## Non-critical

If there was ever a time to try hooks, this might be it. You'll have to do a minor refactor to a functional component, but these lifecycle methods could be replaced with 3 lines. Eventually you'd save even more lines of code 'cos you'd only have to de-structure `this.props` once throughout the scope of the component, instead of for each method.

```
useEffect(() => {
  getLinkedEntitySetBreakdown({ entitySetId, entityKeyIds: [entityKeyId] });
}, [getLinkedEntitySetBreakdown, entityKeyId, entitySetId])
```

Reply:
HOOKS!!!